### PR TITLE
Update parser.md

### PR DESCRIPTION
**Pull request by CommanderXL**

### CODE REVIEW

**Comment by chenxsan** [comment]
Hi, can you please sign the CLA here https://github.com/webpack/webpack.js.org/pull/4113#issuecomment-719874338 so we can get it merged?

**Comment by CommanderXL** [comment]
@chenxsan Well, I have signed the CLA.

**Review by chenxsan — APPROVED** [review]

File: src/content/api/parser.md [hunk 1]
```diff
@@ -532,7 +532,7 @@ Executed when parsing a `MemberExpression`.
 ```js
 const a = process.env;
 
-parser.hooks.new.for('process').tap('MyPlugin', expression => {});
+parser.hooks.expressionAnyMember.for('process').tap('MyPlugin', expression => {});
```
Comment (chenxsan) [inline src/content/api/parser.md:535]: Sorry for the delay. I've checked the [webpack repository](https://github.com/webpack/webpack/pull/9203/files#diff-313038f65941a3a6c139ebe61a484045ace246a429e31975f3606afdd95d977f), seems like `expressionAnyMember` was removed for a long time. So maybe we can remove it at the moment.

**Comment by chenxsan** [comment]
closed in favor of https://github.com/webpack/webpack.js.org/pull/3340

**Pull request closed without merging**